docusaurus/docs/Android/04-compose/02-component-architecture.mdx

```diff
@@ -35,7 +35,9 @@ override fun onCreate(savedInstanceState: Bundle?) {
         ChatTheme { // Theme wrapper
```

Comment:
What do you think about the term `composables` which is used above? Should it start with upper-case letter like `ViewModel`?

Reply:
Done!